### ISSUE TITLE
Recreate cell and header partials when limitation is gone

### ISSUE THREAD

**Issue by chipsenkbeil** [chipsenkbeil/grid-side#19]
Both the cell and header partials for the homepage had to be added back in to the main html template file due to the fact that they needed access to `.Site.BaseURL`. Currently, there is no way to pass additional arguments to a partial and using a partial on a parameter (like `.Site.Params.Header`) results in you losing access to the scratch area.

If this gets resolved, I want to move the contents back out to individual partials.

**Comment by chipsenkbeil** [comment]
Looks like this will be possible in 0.15 via https://github.com/spf13/hugo/pull/1463.

**Comment by dixonge** [comment]
Not sure if this is the right place for this, but on the home page all of my project sections have the baseurl pre-pended in front of the link. But all of these links are to other sites, not internal pages or posts. So the link is broken. I *think* this is coming from line 41 on this page:

https://github.com/chipsenkbeil/grid-side/blob/master/layouts/partials/homepage/single.html

**Comment by chipsenkbeil** [comment]
@dixonge, yep. I realized a couple of days ago that I need to go through all of my links and apply an absolute filter or a relative filter. See [here](http://gohugo.io/templates/functions/#urls:904d9cf85d985550c869e6d9741448cc) for the functions. This will let the links work with external or internal urls instead of what I was doing, which was assume internal.

I'll get to this soon. :)

**Comment by dixonge** [comment]
I just removed the baseUrl code from the home page and will manually include it where needed. That works for me for now, since that page isn't likely to change much.
